esmvaltool/recipes/examples/recipe_check_obs.yml

```diff
@@ -48,7 +48,7 @@ diagnostics:
       rsutcs:
     additional_datasets:
       - {dataset: CERES-EBAF, project: OBS, mip: Amon, tier: 2,
-         type: sat, version: Ed4.0, start_year: 2001, end_year: 2018}
+         type: sat, version: Ed4.1, start_year: 2001, end_year: 2021}
```

Comment:
The recipe does not run successfully unless `version: Ed4.01` is used. Could you please homogenize the version number between the recipe and the cmor_config file?